jwst/photom/tests/test_photom.py

```diff
@@ -290,7 +290,7 @@ def create_input(instrument, detector, exptype,
             input_model.var_poisson = np.ones(shape, dtype=np.float32)
             input_model.var_rnoise = np.ones(shape, dtype=np.float32)
             input_model.var_flat = np.ones(shape, dtype=np.float32)
-            input_model.meta.subarray.name = 'SUBPRISM'     # matches 'GENERIC'
+            input_model.meta.subarray.name = 'FULL'
```

Comment:
Is this supposed to be changed to `SLITLESSPRISM`?

Reply:
No. The actual test case is simulating an LRS fixedslit exposure, which are always taken using SUBARRAY='FULL' (SUBPRISM is only used for slitless). So I don't know why the original was set that way. Apparently it doesn't make any difference to the actual test operations. But I reset the value to 'FULL' just to be consistent with the real params of a LRS fixedslit exposure.

Reply:
I don't think there was any reason why I used SUBPRISM.  It didn't matter, and I just wanted to test the row selection.